test/controllers/systems_controller_test.rb

```diff
@@ -16,10 +16,11 @@ class SystemsControllerTest < ActionDispatch::IntegrationTest
 
   test 'index accepts search' do
     scope = mock('scope')
-    SystemsController.any_instance.expects(:policy_scope).with(Host).returns(scope)
+    SystemsController.any_instance.expects(:policy_scope).with(Host)
+                     .returns(scope)
     scope.expects(:search_for).with('foo=bar')
 
-    get systems_url, params: {search: 'foo=bar'}
+    get systems_url, params: { search: 'foo=bar' }
```

Comment:
I'm really not a fan of this kind of thing in PRs, unless you put it in a separate commit. It's just noise in the review. If you'd like to enforce some of these style things like line length, etc., let's update the rubocop config! I'm fine to keep the changes in this one PR, though.

Reply:
Tests didn't pass if I didn't add this apparently :disappointed: